tools/test-js-with-node

```diff
@@ -63,10 +67,63 @@ if not options.force:
         print('If you really know what you are doing, use --force to run anyway.')
         sys.exit(1)
 
+NODE_COVERAGE_PATH = 'var/node-coverage/coverage.json'
+INDEX_JS = 'frontend_tests/zjsunit/index.js'
+
 os.environ['NODE_PATH'] = 'static'
 os.environ['TZ'] = 'UTC'
 
-INDEX_JS = 'frontend_tests/zjsunit/index.js'
+if options.check_enforced:
+    coverage_json = None
+    print()
+    print("=============================================================================")
+    print("Checking for fully covered files that are not enforced yet...")
+    print("Running `test-js-with-node --coverage` first.")
+    print()
+
+    if args:
+        print('BAD ARGS! Coverage reports run against all files.')
+        sys.exit(1)
+
+    istanbul = os.path.join(ROOT_DIR, 'node_modules/.bin/istanbul')
+    command = [istanbul, 'cover', INDEX_JS, '--dir', 'var/node-coverage']
+
+    try:
+        ret = subprocess.check_call(command)
+    except OSError:
+        print('Bad command: %s' % (command,))
+        raise
+
+    try:
+        with open(NODE_COVERAGE_PATH, 'r') as f:
+            coverage_json = ujson.load(f)
+    except IOError:
+        print(NODE_COVERAGE_PATH + " doesn't exist.")
+        raise
```

Comment:
Should we make test fails if there are files that are fully covered but not enforced? (I'm also guessing that time want be too big of an issue here since there aren't that many lines of code per file)